### ISSUE TITLE
404 from `/process` endpoint when Resource contains a BelongsTo relationship

### ISSUE THREAD

**Issue by gabe565** [dcasia/nova-filepond#6]
Loving this plugin so far, but I am having a strange issue.

Something goes wrong [while trying to load `rulesForCreation`](https://github.com/dcasia/nova-filepond/blob/ddc02c9f4ec30c5f8c335b7901cadaea3996e598/src/Http/Controllers/FilepondController.php#L38) that causes Laravel to return a 404 response.
This only happens when the resource has a `BelongsTo` field which is shown on creation.

**Comment by milewski** [comment]
Can you share how are you using it? what rules did you set

**Comment by gabe565** [comment]
Sure! Sorry, should've given that at first.

The `BelongsTo` itself has no rules, it is defined as:
```
BelongsTo::make('Role')
```

The Filepond config is also pretty simple:
```
Filepond::make('Avatar')
    ->image()
```

The call stack for generating `BelongsTo` rules is really deep, but it seems to actually die in `associatableQueryMethod()` of laravel/nova/src/Fields/BelongsTo.php.
`$request->resource()` is null, so `abort_if(is_null($resource), 404);` is triggered in `resource()` in laravel/nova/src/Http/Requests/InteractsWithResources.php.

**Comment by milewski** [comment]
@gabe565 sorry for the delay, fixed on version `v0.2.8`
